app/common/services/maps.js

```diff
@@ -6,15 +6,18 @@ function Maps(ConfigEndpoint, L, _, CONST) {
         baselayers : {
             satellite: {
                 name: 'Satellite',
+                provider: 'mapbox',
                 url: 'https://api.tiles.mapbox.com/v4/{mapid}/{z}/{x}/{y}.png?access_token={apikey}',
                 layerOptions: {
                     apikey: CONST.MAPBOX_API_KEY,
+                    provider: 'mapbox',
```

Comment:
added this key to avoid guessing by mapid, altough if we set the mapid with the provider.layername always we shouldn't need that. Not sure if this is good or bad yet